tests/python/relay/test_op_level2.py

```diff
@@ -987,6 +1052,8 @@ def test_bitpack_infer_type():
 
 
 if __name__ == "__main__":
+    test_dilation2d_infer_type()
```

Comment:
merge this two together and use test_dilation2d instead

Reply:
@yongwww Reason to have two separate tests is, we are just following conv2d operator convention where there are separate tests for infer type and operator output testing. do you still want to merge test case

Reply:
let's merge them, at least it looks simpler.

Reply:
merged